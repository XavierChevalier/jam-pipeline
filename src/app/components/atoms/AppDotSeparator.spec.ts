import * as Stories from './AppDotSeparator.stories'
import { StorybookTestsGenerator } from '@/tests/tests-generators/StorybookTestsGenerator'

describe('AppDotSeparator', () => {
  StorybookTestsGenerator.fromStoriesExports(Stories).snapshotEachStories()
})
