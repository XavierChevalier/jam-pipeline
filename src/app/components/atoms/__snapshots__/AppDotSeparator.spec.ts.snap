// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppDotSeparator stories snapshots [snapshot] should render the "Default" component story correctly 1`] = `
<span
  class="px-1"
>
  â€¢
</span>
`;
